OpenChange/MAPIStoreContext.m

```diff
@@ -531,17 +529,9 @@ - (uint64_t) idForObjectWithKey: (NSString *) key
   mappingId = [mapping idFromURL: childURL];
   if (mappingId == NSNotFound)
     {
-      //[self warnWithFormat: @"no id exist yet for '%@', requesting one...",
-      //      childURL];
+      [self logWithFormat: @"No id exist yet for '%@', requesting one", childURL];
       mapistore_indexing_get_new_folderID (connInfo->mstore_ctx, &mappingId);
```

Comment:
As we are touching this critical part, it would be great to check the return value of this method to avoid having incorrect values in `mappingId` variable. What do you think?